types/atom.js.flow

```diff
@@ -1804,3 +1812,9 @@ declare class atom$Selection {
     },
   ): string,
 }
+
+declare class atom$LanguageMode {
+ constructor(editor: atom$TextEditor) : void,
+ buffer: atom$TextBuffer,
+ rowRangeForCodeFoldAtBufferRow(row: number): ?Array<number> // Returns an {Array} of the [startRow, endRow]. Returns null if no range.
+}
```

Comment:
Woo thanks for all the declarations.

Though we still need to be careful since we're using a undocumented/unofficial API here 😄